x-pack/plugins/cases/public/components/user_action_tree/user_action_host_isolation_comment_event.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('UserActionHostIsolationCommentEvent', () => {
   it('renders with the correct action and hostname', async () => {
     const wrapper = mount(<HostIsolationCommentEvent {...defaultProps()} />);
     expect(wrapper.find(`[data-test-subj="actions-link-e1"]`).first().exists()).toBeTruthy();
-    expect(wrapper.text()).toBe('isolated host host1');
+    expect(wrapper.text()).toBe('isolate submitted on host host1');
```

Comment:
nit: Using `ISOLATED_HOST` variable would be better to avoid change texts in tests.